openai/__init__.py

```diff
@@ -31,7 +31,8 @@
 organization = os.environ.get("OPENAI_ORGANIZATION")
 api_base = os.environ.get("OPENAI_API_BASE", "https://api.openai.com/v1")
 api_type = os.environ.get("OPENAI_API_TYPE", "open_ai")
-api_version = "2021-11-01-preview" if api_type == "azure" else None
+api_version = "2021-11-01-preview" if api_type in (
+    "azure", "azure_ad", "azuread") else None
```

Comment:
Are these really three different API types? In the other files I only two ApiType values for Azure being used.